data/mods/Xedra_Evolved/effects/effects.json

```diff
@@ -1054,61 +1052,31 @@
       }
     ]
   },
-  {
-    "type": "effect_type",
-    "id": "spell_karma_arms_eff",
-    "name": [ "" ],
-    "desc": [ "" ],
-    "apply_message": "Ethereal, transparent arms grow from your shoulders, and <color_cyan>glow</color>.",
-    "remove_message": "The cyan arms grow dim, and finally disappear.",
-    "rating": "good",
-    "enchantments": [ { "mutations": [ "karma_arms" ] } ]
-  },
-  {
-    "type": "mutation",
-    "id": "stalker_eyes",
-    "name": { "str": "Stalker's Vision" },
-    "points": 1,
-    "visibility": 1,
-    "description": "Your own eyes transform into the orbits of an unknown creature.  You see much better, both in the light and in the dark.",
-    "//": "not actual replacement yet, because i'm afraid of how wearing stuff may interact with it",
-    "valid": false,
-    "starting_trait": false,
-    "purifiable": false,
-    "enchantments": [ { "condition": "ALWAYS", "modified_bodyparts": [ { "gain": "stalker_eyes" } ] } ]
-  },
   {
     "type": "effect_type",
     "id": "stalker_eyes_eff",
-    "name": [ "" ],
-    "desc": [ "" ],
+    "name": [ "Stalker's Vision" ],
+    "desc": [
+      "Your own eyes transform into the orbits of an unknown creature.  You see much better, both in the light and in the dark"
```

Comment:
Remove "own" as it duplicates meaning from the starting "Your".  For the second sentence, "You now see much better in the light and dark." flows better